lib/registries/addon/components/registries-layout/template.hbs

```diff
@@ -0,0 +1,46 @@
+{{yield (hash
+    heading=(component 'registries-layout/block')
+    sidenavGutterClosed=this.sidenavGutterClosed
+    toggleSidenav=(action this.toggleSidenav)
+    metadataGutterClosed=this.metadataGutterClosed
+    toggleMetadata=(action this.toggleMetadata)
+)}}
+
+<div local-class='ContentBackground'>
+    <Gutters
+        @leftMode={{this.sidenavGutterMode}}
+        @leftClosed={{this.sidenavGutterClosed}}
+        @rightMode={{this.metadataGutterMode}}
+        @rightClosed={{this.metadataGutterClosed}}
+        as |gutters|
+    >
+        {{#unless this.hideLeftGutter}}
+            <gutters.left-gutter>
+                {{yield (hash
+                    left=(component 'side-nav'
```

Comment:
Since the left thing is always a nav, let's name it accordingly. `LeftNav`, maybe?